hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl

```diff
@@ -723,6 +754,63 @@ objects:
         - '*'
         verbs:
         - '*'
+    - apiVersion: v1
+      data:
+        ca.crt: '-----BEGIN CERTIFICATE-----
```

Comment:
Is it just me or are there extra newlines here?

Reply:
Source file uses literal style with `|`.. looking at best way to preserve this when adding to the template.

Reply:
Tried dropping the SSS directly as-is (with extra \n) and it syncs correctly to the cluster.  So it's weird to look at, but it's valid and works.